### DISCUSSION TITLE
Make preview editable?

### DISCUSSION THREAD

**Discussion by anshumankomawar-amazon** [ibhagwan/fzf-lua#1476] [Q&A] [answered]
Hi I was hoping on using FzfLua as a picker for a custom plugin. When I populate the list with certain items, I want to show some text associated with each item. I also want this text to be editable before pressing enter on the item. Would this be possible to do with FzfLua? Thank You.

**Comment by ibhagwan** [comment, marked as answer]
https://github.com/ibhagwan/fzf-lua/wiki/Advanced

**Reply by anshumankomawar-amazon** [reply]
Ahh thank you. Will go through this.
